packit_service/service/api/copr_builds.py

```diff
@@ -35,6 +35,13 @@
 ns = Namespace("copr-builds", description="COPR builds")
 
 
+def if_time_exists(datetime_object):
```

Comment:
Would be nice to cover this function with some unit tests, and add type annotation.

Reply:
Sure!